### PR TITLE
updating workshop options for international PD form

### PR DESCRIPTION
**Pull request by annaxuphoto**
## Links

Jira: https://codedotorg.atlassian.net/browse/FND-1722?atlOrigin=eyJpIjoiNzdkNzM1N2M3YTg1NGI2MjllNWZhOTFkNGM0NDU0NjgiLCJwIjoiaiJ9

This PR updates the [international PD form](https://studio.code.org/pd/international_workshop) such that:
1. The fields have been rearranged to move school details higher on the page
2. 2 H4 headings have been added (Tell us about yourself and Tell us about your workshop).
3. Users must now first select a country before selecting partner and facilitator, because they are filtered according to school country.
4. All school information options include a "Not applicable" option
5. The checkbox questions also include a "Not applicable" option

After the changes:

https://user-images.githubusercontent.com/16494556/171304313-ade06f34-eec1-4ef9-8ecd-f7a810ab5c98.mov

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/pd/international_opt_in/InternationalOptIn.jsx [hunk 1]
```diff
@@ -209,10 +221,14 @@ class InternationalOptInComponent extends FormComponent {
     });
 
     const names = cities[selectedCity] || [];
+    let nameOptions = [i18n.pdNotApplicable()];
```
Comment (KatieShipley) [inline apps/src/code-studio/pd/international_opt_in/InternationalOptIn.jsx:224]: I'm curious why we sometimes use this pattern, and sometimes use the unshift pattern?

Comment (annaxuphoto) [reply apps/src/code-studio/pd/international_opt_in/InternationalOptIn.jsx:224]: My thinking was: in the case of departments (or whichever top-level school field), the options depend on school country, which does not have a "Not applicable" choice. So, we add the "Not applicable" option to the front of the list of department options.

For every following field, if the previous field has "Not applicable" selected, the only option available should be "Not applicable." If a different option was chosen, we concatenate the list of additional filtered options. In the event we have only the "Not applicable" option, using `unshift` on the empty array would create something like `['Not applicable', 0]`

Hope this makes sense!

Comment (KatieShipley) [reply apps/src/code-studio/pd/international_opt_in/InternationalOptIn.jsx:224]: It does! Thanks!

**Review by KatieShipley — APPROVED** [review]
🎇

**Review by dju90 — APPROVED** [review]

**Pull request merged**